test/cppsuite/README.md

```diff
@@ -1 +1,132 @@
-# WiredTiger cppsuite
+<div style="text-align: justify">
+
+# Introduction
+The cppsuite is a C++ framework designed to help developers write multithreaded and highly configurable testing for WiredTiger. It is intended to make stress testing as well as scenario-driven testing easy to implement. The framework provides built-in [components](#components) that each offers a set of features to enhance the testing experience and [database operations](#database-operations) with a default implementation that can be overridden at the test level. Each test has a corresponding [configuration file](#test-configuration-file) that defines the workload.
+
+# Table of contents
+- [Introduction](#introduction)
+- [Database operations](#database-operations)
+  * [Populate](#populate)
+  * [Insert](#insert)
+  * [Update](#update)
+  * [Remove](#remove)
+  * [Read](#read)
+  * [Custom](#custom)
+  * [Checkpoint](#checkpoint)
+  * [Validate](#validate)
+- [Components](#components)
+  * [Workload Manager](#workload-manager)
+  * [Operation Tracker](#operation-tracker)
+  * [Timestamp Manager](#timestamp-manager)
+  * [Metrics Monitor](#metrics-monitor)
+- [Test configuration file](#test-configuration-file)
+- [How to execute a test](#how-to-execute-a-test)
+
+# Database operations
+The database operations and their default implementation are described in the subsections below. During a test, a user-defined number of threads can be associated to each of those operations. Currently, the framework only supports operations on keys and values of type string. The values are randomly generated using a uniform distribution.
+
+## Populate
+Populate is the first operation called in a test, before any other operation. It fills the database with an initial dataset.
```

Comment:
We need to put two spaces at the end of a line when we want markdown to display the newline.

Reply:
I removed the extra lines in 5e467ea7a6078c3696f40a23b2955d94d2ee63f7

Reply:
Add new lines in dd2374330134a1b9b8630cb2a915a36883edc8eb